setup.py

```diff
@@ -40,7 +40,8 @@ def get_requirements():
     license="Apache License 2.0",
     install_requires=get_requirements(),
     # pylint: disable=fixme
-    # TODO: Point at MPL main branch once Python MPL is merged into main.
+    # TODO-MPL: Point at PyPI once MPL is released.
+    # This blocks releasing ESDK-Python MPL integration.
```

Comment:
Just wondering do we have a way to enforce this?

Reply:
Yes! PyPI will not let you publish a package that uses GitHub directly. We MUST refer to the published MPL to publish the new ESDK-Python.

Reply:
https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#